### PR TITLE
Add Copyright to NEOVM Files

### PR DESCRIPTION
**Pull request by Liaojinghui**

### CODE REVIEW

**Review by shargon — DISMISSED** [review]
Look good, but maybe redundant with the LICENSE file

**Comment by Liaojinghui** [comment]
> Look good, but maybe redundant with the LICENSE file

copyright and license in the file are not redundant. it's as important as what we have in the root directory.
 
https://opensource.stackexchange.com/questions/292/should-i-include-a-copyright-line-in-every-file

File: src/neo-vm/BadScriptException.cs [hunk 1]
```diff
@@ -1,3 +1,13 @@
+// Copyright (C) 2016-2021 NEO GLOBAL DEVELOPMENT.
```
Comment (erikzhang) [inline src/neo-vm/BadScriptException.cs:1]:
```suggestion
// Copyright (C) 2016-2021 The Neo Project.
```

**Comment by erikzhang** [comment]
Why `KEYS.json` and `VALUES.json` are changed too?

**Comment by Liaojinghui** [comment]
> KEYS.json

honestly, i have no idea, unit test passes on my machine

**Comment by Liaojinghui** [comment]
still has no idea what happened, that two files did not change at all

**Review by erikzhang — APPROVED** [review]

**Review by shargon — APPROVED** [review]

**Pull request merged**